go.mod

```diff
@@ -9,6 +9,7 @@ require (
 	github.com/go-logr/zapr v0.1.1
 	github.com/iancoleman/strcase v0.0.0-20191112232945-16388991a334
 	github.com/kr/text v0.1.0
+	github.com/kubernetes-sigs/kustomize v2.0.3+incompatible // indirect
```

Comment:
Where's this coming from?

Reply:
It is indirect. 2 cents that we can just remove and run go mod tidy and it will no be re-added.

Reply:
that's a good question, I have no idea why/how that's in there, my bad